crates/polars-arrow/src/array/dictionary/mod.rs

```diff
@@ -356,7 +356,7 @@ impl<K: DictionaryKey> DictionaryArray<K> {
 
     /// Returns the keys' value of the [`DictionaryArray`] as `usize`
     /// # Panics
-    /// This function panics iff `index >= self.len()`
+    /// This function panics iff `index >= self.keys.values().len()`
```

Comment:
This is the same thing as `self.len()`, just more convoluted and now involves a private member the caller doesn't know about.